core/tabs/gaming/diablo-ii/d2r-loot-filters.sh

```diff
@@ -4,23 +4,23 @@
 
 # Check for required commands
 for cmd in find curl unzip stty; do
-    if ! command -v "$cmd" >/dev/null 2>&1; then
-        printf "%s\n" "Error: $cmd is not installed."
+    if ! command_exists "$cmd" >/dev/null 2>&1; then
```

Comment:
```suggestion
    if ! command_exists "$cmd"; then
```

The command_exists function pipes output to /dev/null, this is redundant.